.github/workflows/_bazel-build-test.yml

```diff
@@ -71,6 +83,14 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
+      # Apply the filter logic to the build step too if the test-config label is already there
```

Comment:
You don't need to worry about all this `filter-test-configs` step in this PR.  It's not needed and can be removed